assets/js/main.js

```diff
@@ -387,27 +387,28 @@ $(function(){
 });
 // open accordion for direct links to panel id
 $(window).load(function() {
-  var getHash = location.hash; //get hash from js object location
-  function activateAccordion(id) {
-    if (id.length)//check if hash isn't empty
-    {
-      var accordion = $('a[href="' + id + '"]');
-      $panelHeight = accordion.parents('.panel').height();
-      console.log('Offset Top: ' + accordion.parents('.panel').offset().top);
-      $('html,body').animate({
-          scrollTop: accordion.parents('.panel').offset().top //scroll to accordion
-      }, 100, function () {
-          accordion.click(); //simulate click
-          accordion.parents('.panel').find('.panel-title a.no-scroll i.fa').addClass('panel-isOpen'); //point caret down for oepned panel
-      });
-    }
-    else{
-      //if no hash is called, open first panel
-      $('.panel-group .panel:first-child .panel-collapse').addClass('in');
-      $('.panel-group .panel:first-child .panel-title a.no-scroll i.fa').addClass('panel-isOpen');
-    }
-  };
-  activateAccordion(getHash);
+  if(location.pathname.indexOf('/guides') === -1) {
```

Comment:
What happens if we add new pages also without this accordion?

Reply:
I think we can then take off the condition as it will no longer throw an error. Another thought I had was rather what if we have other links that have guides somewhere and has `panel` class. This would be a problem. So I have just updated the code to rather check the value after the base url